### PR TITLE
rm clif submodule; fix master

### PR DESCRIPTION
**Pull request by MarkMcCaskey**
rm cranelift submodule for now, master should be working again after this

### CODE REVIEW

**Comment by MarkMcCaskey** [comment]
bors try

File: .gitmodules [hunk 1]
```diff
@@ -1,7 +1,3 @@
 [submodule "wapm-cli"]
 	path = wapm-cli
-	url = https://github.com/wasmerio/wapm-cli.git
-[submodule "lib/clif-backend/cranelift"]
-	path = lib/clif-backend/cranelift
-	url = https://github.com/wasmerio/cranelift
-	branch = wasmer
+	url = https://github.com/wasmerio/wapm-cli.git
```
Comment (nlewycky) [inline .gitmodules]: end file in newline?

**Pull request merged**